socially/view/data.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+require_once '../controller/Posts.php';
+
+// Takes the starting value from where the next posts will be shown to the user.
+$start = $_POST['starting'];
+$posts = new Posts();
+
+// Loads next 5 posts.
+$data = $posts->fetch_posts($start, 5);
+$output = '';
+foreach ($data as $item => $datum) {
+  $user = $datum['user'];
+  $text = $datum['text'];
+  $post = $datum['post'];
+  $post_time = $datum['post_time'];
+  $pfp = '';
+  if (file_exists("profile_photos/{$user}.jpeg")) {
+    $pfp = "profile_photos/{$user}.jpeg";
+  }
+  else {
+    $pfp = 'assets/user.png';
+  }
+  $output .= '<div class="post">
```

Comment:
If this logic is more load more, can we think of some other way of implementing this instead of storing such long html in php variable